src/tamp_improv/approaches/rl_blocks2d_improvisational_tamp_approach.py

```diff
@@ -110,3 +124,92 @@ def _get_skill_for_operator(
             raise TaskThenMotionPlanningFailure("No skill can execute operator")
         assert len(applicable_skills) == 1, "Multiple operators per skill not supported"
         return applicable_skills[0]
+
+    def _train_policy(self) -> None:
+        """Train the policy if using online training."""
+        if self._train_online and not self._policy_trained:
+            print("\nStarting online policy training...")
+            print(f"Training for {self._train_timesteps} timesteps")
+            self._policy.train(  # type: ignore[attr-defined]
```

Comment:
so perhaps we should extend the interface of `ImprovisationalPolicy` to include not only `get_action`, but also the other methods and attributes that are required in this approach, including `train`, `save`, `load`, and maybe even `env`?